src/screens/learning/components/FlashCardContainer.js

```diff
@@ -77,45 +117,75 @@ const FlashCardContainer = () => {
     }
   }, [list, currentLevel, numQuizzed]);
 
+  const placeHolderText = () => (currentLevel ? NOT_LOGGED_IN : 'All Done!');
+
   return (
-    <View>
-      {!currentLevel ? (
-        <Text>All Done!</Text>
-      ) : (
-        <>
-          <Text>Level {currentLevel}</Text>
-          <FlashCard
-            word={wordInfo}
-            isRevealed={isRevealed}
-            isExpanded={isExpanded}>
-            <Button
-              title={isExpanded ? 'Hide Expressions' : 'Show Expressions'}
-              onPress={handleExpressionsButtonPress}
-            />
-            <Button
-              title={isRevealed ? 'Hide' : 'Reveal'}
-              onPress={handleRevealButtonPress}
-            />
-            {isRevealed ? (
-              <View>
-                <Button
-                  title="Got It"
-                  onPress={() => {
-                    handleCorrectOrIncorrectButtonPress(true);
-                  }}
-                />
-                <Button
-                  title="Missed It"
-                  onPress={() => {
-                    handleCorrectOrIncorrectButtonPress(false);
-                  }}
-                />
-              </View>
-            ) : null}
-          </FlashCard>
-        </>
-      )}
-    </View>
+    <ScreenWrapper>
+      <Container>
+        <Header>
+          <Left />
+          <Body>
+            <Title>
+              {currentLevel ? `Level ${currentLevel}` : 'All Done!'}
+            </Title>
+          </Body>
+          <Right />
+        </Header>
+        <Content>
+          {user && currentLevel ? (
+            <FlashCard
+              word={wordInfo}
+              isRevealed={isRevealed}
+              isExpanded={isExpanded}>
+              <Button
+                small
+                transparent
+                bordered
+                style={styles.button}
+                title={isExpanded ? 'Hide Expressions' : 'Show Expressions'}
+                onPress={handleExpressionsButtonPress}>
+                <Text>
+                  {isExpanded ? 'Hide Expressions' : 'Show Expressions'}
+                </Text>
+              </Button>
```

Comment:
Just looking at your screenshots, I'd suggest putting some space between any two buttons you have side-by-side. Can probably be done a lot of different ways (wrapping them in Views that have padding, using Grid/Col, putting margins directly on buttons, etc)